_includes/login.md

```diff
@@ -1,5 +1,7 @@
 To sign into Quay.io, execute the `docker login quay.io` command:
 
+Note: If you go to your account settings you can create an encrypted password for more cli security.
```

Comment:
Well the account settings page is for whoever is logged in. So not really. For instance https://quay.io/user/<user name>?tab=settings.